### ISSUE TITLE
Doc Accessibility ACC-004 : Insufficient colour contrast for the section symbol (§) within the title (Moderate)

### ISSUE THREAD

**Issue by tfrancart** [sparna-git/shacl-play#323]

**Comment by tfrancart** [comment]
Reported on ReSpec side : https://github.com/speced/respec/issues/5338
